src/test/php/org/bovigo/vfs/visitor/vsfStreamAssertVisitorTestCase.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * This file is part of vfsStream.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @package  org\bovigo\vfs
+ */
+namespace org\bovigo\vfs\visitor;
+use org\bovigo\vfs\vfsStream;
+
+/**
+ * Test for org\bovigo\vfs\visitor\vfsStreamAssertVisitor.
+ *
+ * @since  0.10.0
+ * @see    https://github.com/mikey179/vfsStream/issues/10
+ * @group  issue_10
+ */
```

Comment:
This has nothing to do with issue #10, please remove `@group` and `@see`. Also `@since` should feature the correct version when it was introduced. As this is the feature addition, the next possible version this could be introduced would be 1.6.0.